client/src/components/questions/Question.jsx

```diff
@@ -1,60 +1,84 @@
 /* eslint-disable react/prop-types */
 /* eslint-disable no-unused-vars */
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
+import AnswerList from './AnswerList.jsx';
+import axios from 'axios';
+import API_KEY from '../../../../config.js';
 
-const StyledQuestion = styled.div`
+
+const StyledQandA = styled.div`
   text-align: left;
   padding: 0.7% 0;
 `;
-const StyledAnswer = styled.div`
-  padding: 0.7% 0;
+const StyledQuestion = styled.div`
+  display: flex;
+  justify-content: space-between;
+
 `;
+const StyledLinks = styled.div`
 
-const Question = ({ questionData }) => {
+`;
 
-  // turns the answers object into an array for easy iteration
+// turns the answers object into an array and sorts it based on helpfulness and if the seller answered.
+var sortAnswers = (answersObject) => {
   var answerArray = [];
-  for (var key in questionData.answers) {
-    answerArray.push(questionData.answers[key]);
+  for (var key in answersObject) {
+    answerArray.push(answersObject[key]);
+  }
```

Comment:
If I am reading this correctly, you can use Object.values to automatically create an array of all of the values in an object. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values

Reply:
Good point. Definitely should add that to my repertoire.